src/lib/SX127xDriver/SX127x.cpp

```diff
@@ -146,10 +146,17 @@ void SX127xDriver::SetBandwidthCodingRate(SX127x_Bandwidth bw, SX127x_CodingRate
         //data-sheet errata recommendation http://caxapa.ru/thumbs/972894/SX1276_77_8_ErrataNote_1.1_STD.pdf
         hal.writeRegister(0x36, 0x02, SX12XX_Radio_All);
         hal.writeRegister(0x3a, lowFrequencyMode ? 0x7F : 0x64, SX12XX_Radio_All);
+
+        // 2.3 Receiver Spurious Reception of a LoRa Signal
+        hal.writeRegisterBits(0x31, 0x80, 0x80, SX12XX_Radio_All);
```

Comment:
Why not use the pre-defined `SX127X_REG_DETECT_OPTIMIZE` register value?

Reply:
Might also be worth adding the errata URL as a comment, like we have above?

Reply:
> Why not use the pre-defined `SX127X_REG_DETECT_OPTIMIZE` register value?

Done.

Reply:
> Might also be worth adding the errata URL as a comment, like we have above?

Done.  But Ive linked to Semtech as this is where the most up to date version should be.